docs-v2/integrations/integration-templates/algolia.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Algolia API Integration Template'
+sidebarTitle: 'Algolia'
+---
+
+## Get started with the Algolia template
+
+<Card title="How to use integration templates"
+      href="/understand/concepts/templates"
+      icon="book-open">
+    Learn how to use integration templates in Nango
+</Card>
+
+<Card title="Get the Algolia template"
+      href="https://github.com/NangoHQ/nango/tree/master/integration-templates/teamtailor"
```

Comment:
```suggestion
      href="https://github.com/NangoHQ/nango/tree/master/integration-templates/algolia"
```

Reply:
Thank you Samuel 🙏.This was just a simple sync script to test the integration.